marklogic-data-hub-central/ui/e2e/cypress/integration/curation/curate/addCustomStepToFlow.spec.tsx

```diff
@@ -16,49 +19,39 @@ describe("Add Custom step to a flow", () => {
     cy.contains(Application.title);
     cy.loginAsDeveloper().withRequest();
     LoginPage.postLogin();
-    cy.waitForAsyncRequest();
+    cy.saveLocalStorage();
   });
   beforeEach(() => {
-    cy.loginAsDeveloper().withRequest();
-    cy.waitForAsyncRequest();
-  });
-  afterEach(() => {
-    cy.resetTestUser();
-    cy.waitForAsyncRequest();
+    Cypress.Cookies.preserveOnce("HubCentralSession");
+    cy.restoreLocalStorage();
   });
   after(() => {
-    cy.loginAsDeveloper().withRequest();
     cy.deleteRecordsInFinal(stepName);
     cy.deleteFlows(flowName);
-    cy.resetTestUser();
-    cy.waitForAsyncRequest();
   });
 
   it("Create new flow", () => {
-    cy.waitUntil(() => toolbar.getRunToolbarIcon().should("be.visible")).click();
+    toolbar.getRunToolbarIcon().should("be.visible").click();
     runPage.createFlowButton().click();
-    cy.waitForAsyncRequest();
+
     cy.findByText("New Flow").should("be.visible");
     runPage.setFlowName(flowName);
     runPage.setFlowDescription(`test flow for adding custom step`);
     loadPage.confirmationOptions("Save").click();
-    cy.wait(2000);
-    cy.waitForAsyncRequest();
+
   });
 
   it("Add custom step from Run tile and Run the step", {defaultCommandTimeout: 120000}, () => {
-    cy.waitUntil(() => toolbar.getRunToolbarIcon().should("be.visible")).click({force: true});
+    cy.log("**Expand flow and add step**");
```

Comment:
I noticed a couple of these logs throughout the changes. Is that intended to leave in?

Reply:
This is a great conversation I was hoping to have with you guys. They were intended, I think this practice would be really really helpful for us (this is a great practice I've seen in other projects). Of course not just randomly but through the whole test (This task for existing tests would take a lot of time, that's why I only limited myself to parts were the tests were breaking a lot). I would love to continue this conversation, but coming back to this case. If you think this isolated log does not make sense I can delete it. My intention it's to make debugging and code reading much more clear. Cypress shows this log in the left panel among the rest of the actions, it's a very powerful tool

Reply:
OK sure! If it is intended and a good practice that helps, I think it doesn't hurt to start adding these logs.